components/NewRealmWizard/components/steps/AddNFTCollectionForm.tsx

```diff
@@ -22,27 +21,24 @@ import NFTCollectionModal from '@components/NewRealmWizard/components/NFTCollect
 import { Metaplex } from '@metaplex-foundation/js'
 import { Connection, PublicKey } from '@solana/web3.js'
 import useWalletOnePointOh from '@hooks/useWalletOnePointOh'
-import asFindable from '@utils/queries/asFindable'
 import useLegacyConnectionContext from '@hooks/useLegacyConnectionContext'
 
-function filterAndMapVerifiedCollections(nfts) {
+export function filterAndMapVerifiedCollections(nfts) {
```

Comment:
this only gets used one time, so it should just be colocated with the one time it gets used (wherever you end up putting that). this matters for avoiding circular deps, which the build system usually can cope with but sometimes cant.